draft-bormann-cbor-yang-standin.md

```diff
@@ -282,7 +303,41 @@ of the union, even though it may violate additional constraints outside the sche
 
 ## Defining Stand-In Usage in Schema
 
-TO DO: formally define the YANG extension
+Requiring modifications to a YANG model in order to use it with
+stand-in tags would pose significant deployment hurdles to using
+stand-in tags.
+
+A YANG model may want to restrict the information content in such a
+way that stand-in tags can always be used, e.g., by using date-no-zone
+in place of date where that is applicable, or by excluding features of
+a YANG data type that cannot be represented in a stand-in-tag.
+
+ISSUE: Should this document define such restricted types, e.g.:
```

Comment:
I would say yes. If I'm not missing anything, this applies for the datetime items where the discrepancy between YANG and CBOR is substantial.

Reply:
I think my question was really "should THIS document" do that or do we do that separately, maybe based on a YANG extension to be defined.